system/helpers/AuthHelper.php

```diff
@@ -64,7 +81,20 @@ public function login()
 		//SE EXISTIR ELE LOGA E CRIA A SESSÃO
 		if(count($sql) > 0 AND !empty($sql) )
 		{
-			$_SESSION['user'] = $sql[$this->_userColumn];
+			//VERIFICA SE SÃO VÁRIAS COLUNAS
+			if(is_array($this->_userColumn))
```

Comment:
separa esse bloco em um método separado. tipo um `getUserColumn()` e coloca aqui apenas o `$_SESSION['user'] = $this->getUserColumn(...)`